src/features/home/components/SignUpSignInChoiceModal.tsx

```diff
@@ -0,0 +1,55 @@
+import { t } from '@lingui/macro'
+import { useNavigation } from '@react-navigation/native'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { _ } from 'libs/i18n'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiary } from 'ui/components/buttons/ButtonTertiary'
+import { AppModal } from 'ui/components/modals/AppModal'
+import { Close } from 'ui/svg/icons/Close'
+import { Typo } from 'ui/theme'
+
+interface Props {
+  visible: boolean
+  dismissModal: () => void
+}
+
+const SignUpSignInChoiceModal: FunctionComponent<Props> = ({ visible, dismissModal }) => {
+  const { navigate } = useNavigation()
+
+  function goToSignUp() {
+    dismissModal()
+  }
+
+  function goToLogin() {
+    dismissModal()
+    navigate('Login')
+  }
+
+  return (
+    <AppModal
+      visible={visible}
+      title={_(t`Le pass Culture est accessible à tous ! `)}
```

Comment:
Je ne sais pas si c'est volontaire ou pas
```suggestion
      title={_(t`Le pass Culture est accessible à tous !`)}
```